db/migrate/20160404132250_remove_katello_from_notification_name.rb

```diff
@@ -1,26 +1,8 @@
 class RemoveKatelloFromNotificationName < ActiveRecord::Migration[4.2]
-  class FakeMailNotification < ApplicationRecord
-    self.table_name = 'mail_notifications'
-  end
-
   def up
-    FakeMailNotification.all.each do |notification|
-      if notification_names.keys.include?(notification.name)
```

Comment:
Another approach could be to filter with a `where` (so pushing it to the DB instead of client side). That would be an empty list in most cases and not cause it to break.

Reply:
technically, yeah. but it felt weird to spend time on "fixing" this where it really is not needed anymore.

https://github.com/Katello/katello/pull/10854